build-binutils.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
+
+VERSION=2.38
+
+# Build sed
+curl -LO --insecure https://ftp.gnu.org/gnu/binutils/binutils-$VERSION.tar.xz
+tar xf binutils-$VERSION.tar.xz && cd binutils-$VERSION
+./configure --prefix="${PREFIX}" \
+  --enable-deterministic-archives \
+  --prefix="${PREFIX}" \
+  --disable-werror \
+  --enable-interwork \
+  --enable-multilib \
+  --enable-64-bit-bfd \
+  --enable-targets=all \
+  --disable-nls
+
+make && make install
+cd .. && rm -rf binutils-$VERSION
+
+cd $PREFIX && tar --remove-files -czf $PKGDIR/bootstrap-binutils-$VERSION.tar.gz .
```

Comment:
Looks like some logic could be shared between all these `build-*` scripts?

Reply:
This last step could probably be handled by a "package" script that just takes the package name and version as arguments.